source/_components/yandex_transport.markdown

```diff
@@ -0,0 +1,51 @@
+---
+title: "Yandex transport"
+description: "Instructions on how to setup Yandex transport with Home Assistant."
+logo: yandex.png
+ha_category:
+  - Transport
+ha_release: 0.99
+redirect_from:
```

Comment:
Please remove the `redirect_from`

Reply:
you mean to remove "redirect_from:
 - /components/sensor.yandex_tranport/" , with _- /components/sensor.yandex_tranport/_?

Reply:
yes both lines